python/cugraph/cugraph/gnn/dgl_extensions/cugraph_service_store.py

```diff
@@ -0,0 +1,582 @@
+        _update_feature_map(
+            self.ndata_feat_col_d, feat_name, contains_vector_features, columns
+        )
+        # Clear properties if set as data has changed
+        self.__clear_cached_properties()
+
+    def add_edge_data_from_parquet(
+        self,
+        file_path,
+        node_col_names,
+        src_offset=0,
+        dst_offset=0,
+        canonical_etype=None,
+        feat_name=None,
+        contains_vector_features=False,
+    ):
+        """
+        Add a dataframe describing edge properties to the PropertyGraph.
+
+        Parameters
+        ----------
+        file_path : string
+            Path of file on server
+        node_col_names : string
+            The column names that contain the values to be used as the source
+            and destination vertex IDs for the edges.
+        canonical_etype : string
+            The edge type to be added. This should follow the string format
+            '(src_type),(edge_type),(dst_type)'
+            If not specified, the type of properties will be added as
+            an empty string.
+        src_offset: int,
+            The offset to add for the source node type
+            defaults to zero
+        dst_offset: int,
+            The offset to add for the dst node type
+            defaults to zero
+        feat_name : string or dict {}
+            The feature name under which we should save the added properties
+            (ignored if contains_vector_features=False and the col names of
+            the dataframe are treated as corresponding feature names)
+        contains_vector_features : False
+            Whether to treat the columns of the dataframe being added as
+            as 2d features
+        Returns
+        -------
+        None
+        """
+        c_ar, len_ar = self.client.call_extension(
+            func_name="add_edge_data_from_parquet_remote",
+            file_path=file_path,
+            node_col_names=node_col_names,
+            canonical_etype=canonical_etype,
+            src_offset=src_offset,
+            dst_offset=dst_offset,
+            graph_id=self.gdata._graph_id,
+            result_device=self.device_id,
+        )
+        loaded_columns = _deserialize_strings_from_char_ars(c_ar, len_ar)
+        columns = [col for col in loaded_columns if col not in node_col_names]
+        _update_feature_map(
+            self.edata_feat_col_d, feat_name, contains_vector_features, columns
+        )
+        self.__clear_cached_properties()
+
+    def get_node_storage(self, key, ntype=None, indices_offset=0):
+        if ntype is None:
+            ntypes = self.ntypes
+            if len(self.ntypes) > 1:
+                raise ValueError(
+                    (
+                        "Node type name must be specified if there "
+                        "are more than one node types."
+                    )
+                )
+            ntype = ntypes[0]
```

Comment:
Where does `ntype` get used after this line?

Reply:
Wow, Yup, this was a bug.   I now pass it down here.   https://github.com/rapidsai/cugraph/blob/d24ab529b910687b586d358f62da3fe0b3b95783/python/cugraph/cugraph/gnn/dgl_extensions/feature_storage.py#L146-L151


We previously had a bug in PG which got fixed by this https://github.com/rapidsai/cugraph/pull/2523  but i missed linking it so this slipped through.  


I have added tests here: 
https://github.com/rapidsai/cugraph/blob/d24ab529b910687b586d358f62da3fe0b3b95783/python/cugraph/cugraph/tests/test_dgl_extension_graph_store.py#L391-L423 

From a user perspective it should not matter as node-ids are unique independent of ntype.